api/RTCPeerConnection.json

```diff
@@ -1944,24 +1943,20 @@
           "support": {
             "chrome": [
               {
-                "version_added": "56",
-                "notes": "Promise based version and unprefixed."
+                "version_added": "51",
+                "notes": "Promise based version."
```

Comment:
There's several comments about the introduction of Promises, but they're not worded the same throughout. Can we go with a consistent `"Promise-based version"` perhaps?

Reply:
PTAL